README.md

```diff
@@ -26,14 +26,14 @@ Writing a new text-only article can be very simple using just GitHub:
     Author: Pepa Houbař
     Twitter: houbyfaktzeru
     Gravatar: pepa.houbar@example.com
-    About: Pepa je vášnivý houbař. Má stránky [Houby fakt žeru](http://example.com) a chodí do lesa především kolem Houbové Lhoty. Jeho životním mottem je _vím houby, jím houby_.
+    About: Pepa je vášnivý houbař. Má stránky [Houby fakt žeru](https://example.com) a chodí do lesa především kolem Houbové Lhoty. Jeho životním mottem je _vím houby, jím houby_.
```

Comment:
I don't think https://example.com exists

Reply:
neither does http://example.com :) Or rather, they're the same. I'd rather provide the https:// link in the template.